lab-trent/greet.js

```diff
@@ -0,0 +1,7 @@
+module.exports.greet = function(name) {
```

Comment:
Could totally just be a `module.exports = function() {...}`, rather than a named method.